src/completedAll.test.js

```diff
@@ -0,0 +1,28 @@
+import { tasks, taskPopulate, setStorage } from '../modules/functions.js';
+import removeCompletedTask from './completeAll.js';
+
+// Mock the required functions and objects
+jest.mock('../modules/functions.js', () => ({
+  tasks: [
+    { description: 'Task 1', completed: true },
+    { description: 'Task 2', completed: false },
+    { description: 'Task 3', completed: true },
+  ],
+  taskPopulate: jest.fn(),
+  setStorage: jest.fn(),
+}));
+
+describe('removeCompletedTask', () => {
+  test('Removes completed tasks', () => {
```

Comment:
- Please a test for the function for editing an item's description
- Add another test for a function for updating an items 'completed' status
- Also, test the "Clear All Completed" function

Reply:
hi @afanimartin , we have added those files in modules.
![image](https://github.com/binodbhusal/ToDoList2/assets/116276432/767a0f7b-c4bb-4f1d-853b-db384b053ba0)